blosc2/proxyschunk.py

```diff
@@ -0,0 +1,210 @@
+#######################################################################
+# Copyright (c) 2019-present, Blosc Development Team <blosc@blosc.org>
+# All rights reserved.
+#
+# This source code is licensed under a BSD-style license (found in the
+# LICENSE file in the root directory of this source tree)
+#######################################################################
+import blosc2
+from abc import ABC, abstractmethod
+
+
+class ProxySource(ABC):
+    """
+    Base interface for all supported sources in :ref:`ProxySChunk`.
+    In case the source is multidimensional, the attributes `shape`, `chunks`,
```

Comment:
PEP 8 mandates a blank line between brief description (one line) and the extended descrition.